Core/Extensions/ImageLinkTagHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace WebNetSample.Core.Extensions;
+
+[HtmlTargetElement("webnetsample-imagelink", TagStructure = TagStructure.NormalOrSelfClosing)]
+public class ImageLinkTagHelper : TagHelper
+{
+    [HtmlAttributeName("image-id")]
+    public string? Id { get; set; }
```

Comment:
You should not use `string?` since `string` is a reference type and it is nullable by default

Reply:
Actually, I know but somehow instructor used like this so thats why I used. I'll remove it.